test/unit/web-util-test.js

```diff
@@ -6,8 +6,19 @@ var webUtil = require('../../src/util/web-util')
 test('web-util.composePatchQuery() test', function (t) {
   let toDelLinks = ['triple1', 'triple2', 'triple3']
   let toAddLinks = ['triple4', 'triple5', 'triple6']
-  let expectedQuery = 'DELETE DATA { triple1 . triple2 . triple3 };\n' +
-    'INSERT DATA { triple4 . triple5 . triple6 };\n'
+  let expectedQuery = 'DELETE DATA { triple1 . triple2 . triple3 . };\n' +
+    'INSERT DATA { triple4 . triple5 . triple6 . };\n'
+  t.equal(
+    webUtil.composePatchQuery(toDelLinks, toAddLinks),
+    expectedQuery
+  )
+  t.end()
+})
+
+test('web-util.composePatchQuery() - single statement', function (t) {
+  let toDelLinks = []
+  let toAddLinks = ['triple1']
+  let expectedQuery = 'INSERT DATA { triple1 . };\n'
```

Comment:
@deiu ha! probably. But would you really expect the compose query function to run it through some sort of Turtle validator?